test-engineering/load-tests/README.md

```diff
@@ -39,14 +38,18 @@ directory using the bash script `load_test_setup.sh`.
 Make the file executable by:
 
 ```text
-chmod +x load_test_setup.sh
+chmod +x setup_k8.sh
```

Comment:
I assume `k8` is supposed to stand for Kubernetes here? If so, this should be `k8s`.

See https://github.com/kubernetes/kubernetes#kubernetes-k8s